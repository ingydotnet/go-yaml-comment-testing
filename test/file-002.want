kind: Document
content:
    - kind: Mapping
      style: Flow
      line: '#   comment 4'
      content:
        - kind: Scalar
          head: '#   comment 2'
          text: foo
        - kind: Scalar
          line: '#   comment 3'
          text: bar
kind: Document
foot: '#  comment 5'
content:
    - kind: Scalar
      tag: '!!null'
