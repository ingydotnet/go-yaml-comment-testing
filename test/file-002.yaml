{   #   comment 1
  #   comment 2
  foo: bar    #   comment 3
} #   comment 4

---

#  comment 5
