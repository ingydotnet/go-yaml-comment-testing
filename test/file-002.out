Kind: Document
More:  
    Kind: Mapping
    Styl: Flow
    Line: "#   comment 4"
    More:  
        Kind: Scalar
        Head: "#   comment 2"
        Text: "foo"
        
        Kind: Scalar
        Line: "#   comment 3"
        Text: "bar"
        
Kind: Document
Foot: "#  comment 5"
More:  
    Kind: Scalar
    Tag : !!null
    Text: ""
    
